py-pkgs/08-ci-cd.ipynb

```diff
@@ -721,44 +719,37 @@
     ":name: 08-psr-table\n",
     "| Key    | Description |\n",
     "| :--- | ---: |\n",
-    "|`version_variable`|Location of version number for PSR to bump.|\n",
+    "|`version_toml`|Location of version number for PSR to bump.|\n",
     "|`branch`|Branch where releases should be made from.|\n",
     "|`changelog_file`|Location of changelog file for PSR to update using commit messages.|\n",
     "|`build_command`|How to build new distributions for the release.|\n",
-    "|`dist_path`|Location of distributions after running `build_command`.|\n",
-    "|`upload_to_release`|Whether to automatically create a release of the new package version on GitHub.|\n",
-    "|`upload_to_pypi`|Whether to upload to PyPI. Default is `true`, but we want to upload to TestPyPI first to test things out, so we've turned this off.|\n",
-    "|`remove_dist`|Whether to remove distributions at `dist_path` after upload. We turned this off because we want to upload these distributions to TestPyPI and PyPI ourselves.|\n",
-    "|`patch_without_tag`|Always create a new patch release even if there is no trigger tag, such as \"fix\" or \"feat\", in any commits since the last release.|\n",
     "```\n",
     "\n",
     "With PSR configured in our *`pyproject.toml`*, we can now add it as a step to our CD workflow.\n",
     "\n",
-    "In **{numref}`07:Automatic-version-bumping`**, we used the command `semantic-release version` to get PSR to automatically update our version based on keywords it finds in all the commit messages that have been made since the last tag of your package. In our CD workflow, we'll be using the slightly different command `semantic-release publish`, which will bump our version, update our changelog, tag a new release on GitHub, and build a new sdist and wheel. \n",
+    "In **{numref}`07:Automatic-version-bumping`**, we used the `semantic-release version` to get PSR to automatically update our version based on keywords it finds in all the commit messages that have been made since the last tag of your package. In our CD workflow, we'll instead be using GitHub Actions to run two tasks using PSR, the first action, [python-semantic-release/python-semantic-release@v8.3.0](https://python-semantic-release.readthedocs.io/en/latest/github-action.html), will bump our version, update our changelog, tag a new release on GitHub, and build a new sdist and wheel. We will discuss this now. Later we will a second PSR GitHub action, [python-semantic-release/upload-to-gh-release@main](https://github.com/python-semantic-release/upload-to-gh-release), to also create a release on GitHub.\n",
```

Comment:
Missing word discuss/show: at "... Later, we will _discuss_ a second PSR Github..."